src/main/java/models/Suit.java

```diff
@@ -6,6 +6,6 @@
  * Created by michaelhilton on 1/25/16.
  */
 public enum Suit  implements Serializable {
-    Hearts, Spades, Diamonds, Clubs
+    Hearts, Spades, Diamonds, Clubs, Sword, Cups, Coins
```

Comment:
Sword -> Swords

You call plural "Swords" in your code, but it's been declared as singular in the suit class.